src/test/java/org/jenkinsci/plugins/workflow/steps/ArtifactArchiverStepTest.java

```diff
@@ -41,6 +41,8 @@ public void archive() throws Exception {
         VirtualFile archivedFile = b.getArtifactManager().root().child("msg.out");
         assertTrue(archivedFile.exists());
         assertEquals("hello world", IOUtils.toString(archivedFile.open()));
+        j.assertLogContains("The 'archive' step is deprecated and may not work properly, please use 'archiveArtifacts' instead.",
```

Comment:
If you are going to assert a user message, put it in `Message.properties`.

(There is a longstanding question whether strings used in build logs should be localized, since the user’s native language may not match the locale of the server, but merely creating an English key harms nothing.)